kura/org.eclipse.kura.linux.systemd.provider/src/main/java/org/eclipse/kura/internal/linux/systemd/net/dns/LinuxDnsServerSystemD.java

```diff
@@ -12,11 +12,33 @@
  *******************************************************************************/
 package org.eclipse.kura.internal.linux.systemd.net.dns;
 
+import java.io.File;
+
+import org.eclipse.kura.KuraErrorCode;
+import org.eclipse.kura.KuraException;
 import org.eclipse.kura.internal.linux.net.dns.DnsServerService;
 import org.eclipse.kura.linux.net.dns.LinuxDnsServer;
 
 public class LinuxDnsServerSystemD extends LinuxDnsServer implements DnsServerService {
 
+    private static final String BIND9_COMMAND = "bind9";
+    private static final String NAMED_COMMAND = "named";
+    private static final String BIND9_SERVICE_UNIT_LOC = "/lib/systemd/system/named.service";
+    private static final String NAMED_SERVICE_UNIT_LOC = "/etc/systemd/system/bind9.service";
```

Comment:
The exception I get is because bind9 is in /lib/systemd.
In any case it seems that probably we should choose a different way to get the list of services. Maybe directly invoking systemctl?